config/default_settings.yaml

```diff
@@ -0,0 +1,17 @@
+# Default arguments used for the CLI. Can be overridden using the corresponding
+# flag at runtime
+default_args:
+  run_github_workflow: True
+  run_glue_crawler: True
+  upload_logs: True
+  upload_data: True
+
+# Data type overrides are necessary to maintain parity/consistency with sqoop.
+# Schema overrides are defined in three places, with each overriding the last:
+#   1. By default, NUMBER Oracle types are converted to DECIMAL(10,0)
+#      and TIMESTAMP Oracle types are converted to STRING
+#   2. Global schema overrides, as defined below
+#   3. Table schema overrides, defined in table_definitions.yaml
+global_schema_overrides:
+  - deactivat: STRING
+  - taxyr: STRING
```

Comment:
**[Suggestion, non-blocking]** Unless we imagine that it would be useful to allow a user to configure multiple overrides for one field, it might be cleaner to format this attribute as a key-value mapping rather than a list of key-value mappings:

```suggestion
global_schema_overrides:
  deactivat: STRING
  taxyr: STRING
```

Reply:
Good thinking! Also lets us drop a helper function. Fixed in 770c37b.